manifests/e/Evdial/MikuMod/21/Evdial.MikuMod.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Evdial.MikuMod
+PackageVersion: 21
+PackageLocale: en-US
+Publisher: EvDial
+PackageName: Hatsune Miku Modification
+License: Apache 3
+ShortDescription: 3 original songs, 6 bonus songs, cutscenes, cool menu shit

```

Comment:
```suggestion
ShortDescription: 3 original songs, 6 bonus songs, cutscenes, cool menu
```

Reply:
this appears to be the official description, not one I made